packages/selection/frontend/sirius-components-selection/src/useSelectionDescription.types.ts

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+export interface UseSelectionDescriptionProps {
+  editingContextId: string;
+  selectionDescriptionId: string;
+  targetObjectId: string;
+}
+
+export interface UseSelectionDescriptionValue {
+  loading: boolean;
+  selectionDescription: SelectionDescription | null;
+}
+
+export interface GetSelectionDescriptionVariables {
+  editingContextId: string;
+  representationId: string;
+  targetObjectId: string;
+}
+
+export interface SelectionDescription {
```

Comment:
GQLSelectionDescription to keep indicating that this type comes from the backend

Reply:
Done